sphinx/builders/epub3.py

```diff
@@ -157,30 +169,23 @@ def build_navlist(self, nodes):
                 continue
             if node['level'] > self.config.epub_tocdepth:
                 continue
-            if node['level'] == level:
-                navlist.append(self.new_navlist(node, level))
-            elif node['level'] == level + 1:
-                navstack.append(navlist)
-                navlist = []
-                level += 1
-                if lastnode and self.config.epub_tocdup:
```

Comment:
I think epub_tocdup is not needed in epub3.

epub builder reads toc like this:

```
* Parent
  * Child 1
  * Child 2
```

then generates toc like this:

```
* Parent
   * Parent
   * Child 1
   * Child 2
```

It feels strange for me. So I think it should be omitted in future release. Epub3 builder is an experimental one and it should have clean result (and now, epub3 builder abort when epub_tocdup = False).